dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/AbstractFullReadOnlyController.java

```diff
@@ -396,7 +397,7 @@ else if ( (property.isSimple() || property.itemIs( PropertyType.REFERENCE ))
 
         List<String> fields = Lists.newArrayList( contextService.getParameterValues( "fields" ) );
         List<String> filters = Lists.newArrayList( contextService.getParameterValues( "filter" ) );
-        forceFiltering( filters );
+        forceFiltering( new WebOptions( rpParameters ), filters );
```

Comment:
`rpParameters` is not a clear var name to me (I assume the prefix rp  = request parameter ; with again parameters as suffix). What do you think about simply calling the var `parameters`.

Reply:
Its an existing var name used throughout that class. I did'nt want to rename the var name for the whole class. But we could call it "Request Path Parameters" if that makes more sense for rpParameters 😃